netlogo-gui/src/main/app/HelpActions.scala

```diff
@@ -10,9 +10,9 @@ import javax.swing.{ Action, AbstractAction }
 
 import org.nlogo.core.I18N
 import org.nlogo.api.Version
-import org.nlogo.swing.{ BrowserLauncher, UserAction },
-  BrowserLauncher.docPath,
-  UserAction._
+import org.nlogo.swing.{ BrowserLauncher, UserAction }
```

Comment:
Why did you remove the comma here? It is necessary to indicate that the following two inputs go with `org.nlogo.swing`, and the way it was before matches the import style of the rest of the repo.

Reply:
IntelliJ's import functionality is incredibly frustrating with respect to the spacing of braces and to combining imports. I'll revert that now